components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/util/SAMLSSOUtil.java

```diff
@@ -1432,6 +1432,58 @@ public static boolean validateACS(String tenantDomain, String issuerName, String
         }
 
     }
+
+    /**
+     * Get the Default ACS URL from the service provider config.
+     *
+     * @param tenantDomain
+     * @param issuerName
+     * @return
+     * @throws IdentityException
+     */
+    public static SAMLSSOServiceProviderDO getSPConfig(String tenantDomain, String issuerName) throws
```

Comment:
Let's move this to a private method since this is used in only one class. This will prevent any API additions.